docs/Contribute.md

```diff
@@ -0,0 +1,126 @@
+**Contribution Guidelines**
+
+Cromwell is an open-source project and we heartily welcome community contributions to both our code and our documentation. Here are some guidelines for adding documentation and recommendations on where we could use help the most. 
+
+First off, here are useful links:
+
+* Cromwell documentation: [cromwell.readthedocs.io/en/develop](http://cromwell.readthedocs.io/en/develop)
+* Source on Github: [github.com/broadinstitute/cromwell](https://github.com/broadinstitute/cromwell/tree/develop/docs)
+* Builds on ReadTheDocs: [readthedocs.org/projects/cromwell](https://readthedocs.org/projects/cromwell/builds/)
+* How to build and view the documentation locally: [mkdocs.readthedocs.io](https://mkdocs.readthedocs.io/en/stable/#installation)
+
+### "needs docs"
+
+There are plenty of areas of the Cromwell documentation that need to be updated, improved, or added. Within the Cromwell repo on Github there are many issues labeled as ["needs docs"](https://github.com/broadinstitute/cromwell/issues?q=is%3Aopen+is%3Aissue+label%3A%22needs+docs%22), so feel free to start there. 
+
+If you would like to request additional documentation, you can create a [Github issue in the Cromwell repo](https://github.com/broadinstitute/cromwell/issues/new).
+
+### Writing Tips
+
+1. Keep it clear, accurate, and concise.
+2. Put the most important information first.
+3. Use the second person, use “you” instead of “the user”.
+4. No passive verbs (everything is done by something).
+5. Link to the original source, don't repeat documentation. 
+
+### Formatting
+
+The documentation is written in Markdown. Click here for a [Github Guide on Markdown](https://guides.github.com/features/mastering-markdown/), and click here for more [tips from MkDocs](http://www.mkdocs.org/user-guide/writing-your-docs/).
+
+### Styling
```

Comment:
Maybe have a link to a more complete github markdown page? Otherwise is this going to be a slightly arbitrary subset?

Reply:
As far as I can tell, this is the complete Github Guide to Markdown page. It has all of the Markdown syntax and the Github-flavored syntax.

Reply:
So just OOI is there a reason for duplicating it here instead of linking?

Reply:
I understand you now. The reason I am duplicating here is that there is some ReadTheDocs-specific Markdown syntax for styling links, code, etc.